includes/Core/Assets/Assets.php

```diff
@@ -433,23 +433,23 @@ private function get_assets() {
 			new Script(
 				'googlesitekit-activation',
 				array(
-					'src'          => $base_url . 'js/googlesitekit-activation.js',
```

Comment:
We should keep these `src` arguments around I'd say. It will be overridden internally by the `Script` class, and having them here makes it more obvious what file it actually is.

Reply:
Reverted.